README.rst

```diff
@@ -162,32 +162,28 @@ Transformer Engine comes preinstalled in the pyTorch container on
 From source
 ^^^^^^^^^^^
 
-First, install the prequisites.
+For JAX and Tensorflow, you need to install that dependency first:
 
 .. code-block:: bash
 
-  apt-get install ninja-build pybind11-dev
+  pip install pybind11
 
-Clone the repository and inside it type:
+Then, you can install this optional dependency:
```

Comment:
```suggestion
Then, the following optional dependencies can be installed:
```

Reply:
Stating the purpose of these dependencies would be useful, since they are 'optional'.